h2o-core/src/main/java/water/util/MRUtils.java

```diff
@@ -223,6 +223,9 @@ public static Frame sampleFrameStratified(final Frame fr, Vec label, Vec weights
       numrows += sampling_ratios[i] * dist[i];
```

Comment:
also we could add detection of NaN here or even sooner where one of these factors becomes NaN

Reply:
yeah, we could - but IMHO not super useful, reporting ex-post seems enough to me - it is not expected this math will fail at this point